src/app/(app)/missing-shoes/actions/create-missing-shoes-report.ts

```diff
@@ -0,0 +1,22 @@
+"use server";
+import { redirect } from "next/navigation";
+import { Routes } from "@/lib/routes";
+import { MissingShoeStatus } from "@/constant/constant";
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { MissingSchemaType } from "../add/components/MissingShoesFormBuilder";
+
+export const createMissingShoesReport = async (values: MissingSchemaType) => {
```

Comment:
Can we rename this to be `reportMissingShoe`? Also, it's a single shoe, right?

Reply:
Yes here we are adding shoes so I have rename this